tests/test-sources/plugins/languages/treesitter/treesitter.nix

```
@@ -120,4 +120,20 @@
       ];
     };
   };
+
+  disable-init-selection = {
+    plugins.treesitter = {
+      enable = true;
+
+      settings = {
+        incremental_selection = {
+          enable = true;
+
+          keymaps = {
+            init_selection = false;
+          };
```

```suggestion
          keymaps.init_selection = false;
```

(Up to you, I don't mind either syntax)